test/y2storage/filesystems/type_test.rb

```diff
@@ -231,4 +231,27 @@
       end
     end
   end
+
+  describe "label_valid_chars" do
+    it "contains alphanumeric characters" do
+      chars = Y2Storage::Filesystems::Type::EXT2.label_valid_chars
+      expect(chars).to match(/ABCDEFG/)
+      expect(chars).to match(/xyz/)
+      expect(chars).to match(/789/)
+    end
+
+    it "contains some non-alphanumeric characters" do
+      chars = Y2Storage::Filesystems::Type::EXT2.label_valid_chars
+      expect(chars).to include("-")
+      expect(chars).to include("_")
+      expect(chars).to include(".")
+    end
+
+    it "does not contain any whitespace" do
+      chars = Y2Storage::Filesystems::Type::EXT2.label_valid_chars
+      expect(chars).not_to include(" ")
```

Comment:
you can in fact write `expect(chars).not_to match(/\s/)` and it will check that not whitespace is there.

Reply:
just NP